src/tictactoe/assets/js/app.js

```diff
@@ -3,12 +3,16 @@ import ReactDOM from 'react-dom'
 
 import GameMenu from './components/GameMenu'
 import Board from './components/Board'
+import Footer from './components/Footer'
 
 const Main = () => 
     <div>
-        <h1>Tic Tac Toe will be here soon!</h1>
-        <GameMenu />
-        <Board />
+        <header>
```

Comment:
This feel like odd semantic HTML.
I think I'd be fine with the h1 being in the header, but it also fine to leave the h1 alone.

According to https://developer.mozilla.org/en-US/docs/Glossary/Semantics <header> tends to contain "introductory content and navigation".

Reply:
I reused `<header>` just to have a nice line from it's css:)
Will draw this line differently, indeed no need to reuse this tag here